patches/chrome-installer-linux-rpm-additional_deps.patch

```diff
@@ -0,0 +1,11 @@
+diff --git a/chrome/installer/linux/rpm/additional_deps b/chrome/installer/linux/rpm/additional_deps
+index ea05c3b2ef4e8..f7d62f920f4f7 100644
+--- a/chrome/installer/linux/rpm/additional_deps
++++ b/chrome/installer/linux/rpm/additional_deps
+@@ -44,3 +44,6 @@ libcurl.so.4()(64bit)
+ 
+ # For OS integration.
+ xdg-utils
++
++# Try and use GeoClue2 for Location services
++geoclue2.x86_64
```

Comment:
@rillian Could you please review this line and confirm it works fine on Fedora?

Reply:
(fwiw @rillian I have no idea what I'm doing)

Reply:
see comments above for `additional_deps`

Reply:
also what about arm/x86?

Reply:
This should just be `geoclue2` for rpm-based distros.

We don't want an architecture suffix because support both `x86)64` and `aarch64`. Generally if someone manages to build a package we want it to use the native library whatever the target platform is. Listings often include the suffix for clarity, but it's normally stripped before adding a name to a requirements list.